### DISCUSSION TITLE
Profile picture beside wallet address after connect

### DISCUSSION THREAD

**Discussion by rickywong2207** [rainbow-me/rainbowkit#1775] [Feedback]
hi, i want to ask, if i use < ConnectButton >, there will be like profile picture beside wallet address, but if i use <ConnectButton.Custom>, the profile picture disappear, it not shown in custom connect button.
Can you teach me how to show/call the profile picture when i use custom connect button?
Thank you!

**Comment by 0xmkh** [comment]
@rickywong2207 Would you like to show what exactly is not working ? With `<ConnectButton.Custom>` you can return any elements you would like including the `<img />` element to show your profile picture, but with pure `<ConnectButton />` it won't work.

[Here](https://www.rainbowkit.com/docs/custom-connect-button) is an example of how you can return any elements to your `<ConnectButton.Custom>`.

**Reply by rickywong2207** [reply]
![Screenshot 2024-02-19 at 20 26 37](https://github.com/rainbow-me/rainbowkit/assets/156171552/51ebabf9-2759-4350-b116-a174edbce6f0)
![Screenshot 2024-02-19 at 20 26 54](https://github.com/rainbow-me/rainbowkit/assets/156171552/d22fe00b-a376-45d4-a494-3fa705213066)
![Screenshot 2024-02-19 at 20 27 03](https://github.com/rainbow-me/rainbowkit/assets/156171552/65aa9a50-7c9c-4659-8dc1-a885ffd70163)

if i use default <ConnectButton> there is avatar beside wallet address, there's picture that i attach that i used <connectButton.Custom> but the avatar do not shown, is there a way to make the avatar appear?

**Reply by 0xmkh** [reply]
Ah you're talking about the avatar. You can't get that same avatar unless you create your own one in `<ConnectButton.Custom>`. The `<ConnectButton.Custom>` component exposes only props and not styles.

**Reply by rickywong2207** [reply]
oh so i cannot get the default avatar if i use <ConnectButton.Custom>? okay, thank you!

**Reply by 0xmkh** [reply]
You can get it by accessing the `account.ensAvatar` if that's what you mean ? That's the ens avatar.

**Reply by rickywong2207** [reply]
but when i use account.ensAvatar, it show nothing. it's like no image fetched.

**Reply by 0xmkh** [reply]
Right. That can happen sometimes if you don't have an ens profile image. Do you have an ens name ?

**Reply by rickywong2207** [reply]
i think i never create profile in ens, so it doesn't show anything. i can check it in here right?

https://app.ens.domains/

**Reply by 0xmkh** [reply]
Yes you can check in there correct. [Here](https://app.ens.domains/vitalik.eth) is an example of vitalik's ens name. He has a profile picture which means you can get that from him if he connects the wallet.

Also a good tool to use is called [impersonator](https://www.impersonator.xyz/) where you can connect to wallets with read-only and see if you can get the profile picture from an ens name that has one.

**Reply by rickywong2207** [reply]
okay, thank you so much for your help! cheers!!
